app/controllers/application.js

```diff
@@ -1,9 +1,16 @@
 import Controller from '@ember/controller';
-import { not } from '@ember/object/computed';
+import { not, and } from '@ember/object/computed';
+import config from 'incremental-empire/config/environment';
 
 export default Controller.extend({
-  isManaHidden: not('model.manaUnlocked'),
-  isCultureHidden: not('model.cultureUnlocked'),
-  isMoneyHidden: not('model.moneyUnlocked'),
-  isScienceHidden: not('model.scienceUnlocked'),
+  prod: !config.APP.DEBUG,
+
+  isManaLocked: not('model.manaUnlocked'),
+  isManaHidden: and('isManaLocked', 'prod'),
+  isCultureLocked: not('model.cultureUnlocked'),
+  isCultureHidden: and('isCultureLocked', 'prod'),
+  isMoneyLocked: not('model.moneyUnlocked'),
+  isMoneyHidden: and('isMoneyLocked', 'prod'),
+  isScienceLocked: not('model.scienceUnlocked'),
+  isScienceHidden: and('isScienceLocked', 'prod'),
```

Comment:
Use Unlocked/Displayed instead of Locked/Hidden, to be coherent with empire and have a debug fiels instead of "prod"